index.html

```diff
@@ -1,10 +1,170 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="description" content="IT scholl from Belarus">
+    <meta nam="keywords" content="HTML,CSS,JavaScript,React">
+    <meta name="avtor" content="SamonchikAleksandr">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Lesson 3</title>
+</head>
+<body>
+<h1>Hello word</h1>
+<h2>Image</h2>
+<img src="./img/stitch.jpg" alt="Stitch">
+<h2>List</h2>
+<ul>
+    <li>
+        1.Знакомство с Web.
+    </li>
+    <li>
+        Краткая история веб и что такое интернет.
+    </li>
+    <li>
+        Как устроен браузер и как он получает страницы по сети.
+    </li>
+    <li>
+        Что такое Git?
+    </li>
+    <li>
+        Создание первого репозитория.
+    </li>
+    <li>
+        О файле .gitignore.
+    </li>
+</ul>
+</ul>
+<ul>
+    <li>
+        2.HTML-верстка-первые шаги.
+    </li>
+    <ul>
+        <li>Настройка рабочей среды.</li>
+        <li>Краткая история HTML.</li>
+        <li>Какие бывают теги и что такое атрибут.</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+    </ul>

```

Comment:
`<ul>` shouldn't be a child of `<ol>`. But it could be a child of `<li>`.
Read about [context in which this element can be used](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element).